src/components/routes/Trips/ActiveTripCard.tsx

```diff
@@ -24,23 +23,24 @@ interface Props {
 
 const ActiveTripCard = ({ onCancelClick, trip }: Props) => {
   const { checkInDate, checkOutDate, id, listing, status } = trip;
+  const { lat, lng } = listing;
   const streetAddress = (listing.addressLine1 || '').concat(listing.addressLine2 ? `, ${listing.addressLine2}` : '');
   const isApproved = status === 'host_approved';
   const isStarted = status === 'started';
   const displayStatus = getUserBookingDisplayStatus(status);
   const titleLink = status === 'started' ? `/bookings/${id}/options` : `listings/${listing.idSlug}`;
   return (
-    <ActiveTripCardContainer className="bee-active-trip-card">
+    <ActiveTripCardContainer className="active-trip-card">
       <div className="active-trip-photo">
         <LazyImage src={listing.listingPicUrl} transition />
       </div>
-      <div className="trip-large-section">
+      <div className="active-trip-info">
         <h3>
           <BeeLink to={titleLink}>{listing.title}</BeeLink>
         </h3>
         <div className="bee-flex-div" />
         <div className="address">
-        <BeeLink href={getGoogleMapURI(listing)} target="_blank">
+        <BeeLink href={`https://www.google.com/maps/search/?api=1&query=${lat},${lng}`} target="_blank">
```

Comment:
i think keeping the `getGoogleMapURI` method is sensible and just changing the address to lat/lng.

Reply:
reinstated `getGoogleMapURI` and put it in `utils/formatter` since it is used in both `StartedTripCard` and `ActiveTripCard`